projects/RPi/options

```diff
@@ -144,6 +144,7 @@
 
   # disable some cores which are unuseful for RPi
     LIBRETRO_CORES="${LIBRETRO_CORES//4do/}"
+    LIBRETRO_CORES="${LIBRETRO_CORES//beetle-bsnes_hd/}"
```

Comment:
beetle-bsnes_hd doesn't exist, does it?

Reply:
ah... that should have been only bsnes_hd... lemme fix that...

Reply:
Why did you disable it btw?

Reply:
I think we should change the way we create the `LIBRETRO_CORES` list to avoid removal/substitution of partial core names (mame{-*} is nice example). I would propose something like:
```
LIBRETRO_CORES="\
 corename \
 corename1 \
 corename2 \
 corename3 \
"
```
It would also make easier the reading of the changes when new core is added to the list / cores are removed from the list (now it only shows that the line changed, but not what was added/removed).

When disabling core(s) per target we would then use:
```
LIBRETRO_CORES="${LIBRETRO_CORES//\ corename\ /}"
```
so we match the core including leading and trailing space.

In above example, if you want to disable `corename` and you would use
```
LIBRETRO_CORES="${LIBRETRO_CORES//corename/}"
```
your list would be
```
   1  2  3 
```
But using
```
LIBRETRO_CORES="${LIBRETRO_CORES//\ corename\ /}"
```
matches including the spaces, so the list would be:
```
 corename1  corename2  corename3 
```